apps/common/models.py

```diff
@@ -305,3 +305,15 @@ def type_enum(self):
     def usable_objects(cls):
         """Get objects that are mot marked for deletion."""
         return cls.objects.filter(marked_as_deleted=False)
+
+
+class Announcement(UserResource, FirebasePushResource):  # type: ignore[reportIncompatibleVariableOverride]
```

Comment:
Not sure if this should be added on common/models
cc: @thenav56 ?

Reply:
@tnagorra i didnot found any app relevant so i kept it in common/models.